### PR TITLE
release prep for Mrinversion v0.3.0

### PR DESCRIPTION
**Pull request by deepanshs**

### CODE REVIEW

File: docs/installation/install.rst [hunk 1]
```diff
@@ -0,0 +1,65 @@
+============
+Installation
+============
+
+Requirements
+------------
+
+``mrinversion`` has the following strict requirements:
+
+- `Python <https://www.python.org>`_ 3.6 or later
```
Comment (mgiammar) [inline docs/installation/install.rst:10]: If I remember correctly, mrsimulator requires Python >= 3.7 now

Comment (deepanshs) [reply docs/installation/install.rst:10]: yeah. you are correct. We should also update the requirements file. Would you be up for updating this PR?

Comment (mgiammar) [reply docs/installation/install.rst:10]: Yes I can do that

Comment (deepanshs) [reply docs/installation/install.rst]: updated the python and numpy version

**Review by mgiammar — APPROVED** [review]

**Review by mgiammar — APPROVED** [review]

**Pull request merged**